### ISSUE TITLE
[BUG]  E: Unable to locate package u"nagios-plugins-contrib"

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack-formulas/nagios-formula#73]
### Formula commit hash / release tag

fb96ae05ca094d5a5a3405d10a96fbfc762d1fea

---

### Describe the bug

Details of the bug are upstream https://github.com/saltstack/salt/issues/55620 notifying here in case this is not an upstream bug

All other sections of the bug report are on that issue.

**Comment by myii** [comment]
@arthurlogilab This will almost definitely be resolved by appending `| json` to the following line:

https://github.com/saltstack-formulas/nagios-formula/blob/fb96ae05ca094d5a5a3405d10a96fbfc762d1fea/nagios/plugins.sls#L10

```sls
    - pkgs: {{ salt['pillar.get']("nagios:plugins:extra_pkgs") | json }}
```

Please let us know if that does the trick for you.

**Comment by myii** [comment]
FYI: https://docs.saltstack.com/en/latest/topics/releases/2019.2.0.html#non-backward-compatible-change-to-yaml-renderer.

We're not using `| tojson` just yet because we're still supporting older version of Salt that don't have that.  `| json` works out fine for the time being.

**Comment by myii** [comment]
Fixed by #75.

**Comment by myii** [comment]
@arthurlogilab You'll probably want to close the upstream issue: https://github.com/saltstack/salt/issues/55620.
